### PR TITLE
feat: Add bundle visualizer

### PR DESCRIPTION
**Pull request by roninjin10**
Adds a recomended setup for measuring bundle size.

- Uses vite because the repo is already using vitest
- Uses `safest` option for the most conservative worst case tree shaking
- Adds a visualizer for visualizing bundle size

Run `npm run visualize:bundle` to vie a visualization of bundle for evm package

To test a specific import modify the index.ts to only export that import

image

### CODE REVIEW

File: packages/evm/vite.config.ts [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+  plugins: [wasm(), topLevelAwait(), visualizer({ open: true, gzipSize: true })],
+  build: {
+    rollupOptions: {
+      // We choose safest to get worst case values
+      treeshake: 'safest',
+    },
+    lib: {
+      entry: 'src/index.ts',
+      name: '@ethereumjs/evm',
+      fileName: (format) => `my-library.${format}.js`,
```
Comment (roninjin10) [inline packages/evm/vite.config.ts:16]:
```suggestion
      fileName: (format) => `ethereumjs-evm-bundle.js`,
```

**Review by acolytec3 — APPROVED** [review]
This is super neat.

**Comment by roninjin10** [comment]
Btw you can manually play around with tree shaking via changing index.ts to export only what you want to measure and see how big the difference is vs measuring everything

**Comment by holgerd77** [comment]
Ok, yeah, that's super great! Will merge!

**Pull request merged**